js/documents.js

```diff
@@ -771,6 +774,9 @@ var documentsMain = {
 		if (documentsMain.returnToDir) {
 			documentsMain.overlay.documentOverlay('show');
 			window.location = OC.generateUrl('apps/files?dir={dir}', {dir: documentsMain.returnToDir}, {escape: false});
+		} else if (documentsMain.returnToShare) {
+			documentsMain.overlay.documentOverlay('show');
+			window.location = OC.generateUrl('s/{shareToken}', {shareToken: documentsMain.returnToShare}, {escape: false});
```

Comment:
in general, use OC.redirect() as it's mockable in JS tests.
I guess there are no JS tests here ?

Reply:
There are no integration tests yet, yes.